site/content/installation/_index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "Installation"
+description: 
+weight: 200
```

Comment:
On the main ToC, installation appears below How-To guides. I think it makes more sense to have Installation show up first, since that's the first step.

Reply:
This was a regression. Installation was supposed to be on top of how-tos. It has been resolved now